docs/en_us/internal/testing.rst

```diff
@@ -197,46 +208,58 @@ There is also a ``--fail_fast``, ``-x`` option that will stop nosetests
 after the first failure.
 
 common/lib tests are tested with the ``test_lib`` task, which also
-accepts the ``--failed`` and ``--fail_fast`` options. For example::
-
-    paver test_lib -l common/lib/calc
-
-or
+accepts the ``--failed`` and ``--fail_fast`` options.
 
 ::
 
+    paver test_lib -l common/lib/calc
     paver test_lib -l common/lib/xmodule --failed
 
-To run a single nose test file::
+For example, this command runs a single nose test file.
+
+::
 
     nosetests common/lib/xmodule/xmodule/tests/test_stringify.py
 
-To run a single nose test::
+And this command runs a single nose test within the file.
+
```

Comment:
I'd leave off the "And", maybe "This command runs a single nose test within a specified file."